cmd/create/cluster/cmd.go

```diff
@@ -3258,8 +3259,8 @@ func clusterConfigFor(
 
 func validateBillingAccount(billingAccount string) error {
 	if billingAccount == "" || !ocm.IsValidAWSAccount(billingAccount) {
-		return fmt.Errorf("Billing account is invalid. Run the command again with a valid billing account. %s",
-			listBillingAccountMessage)
+		return fmt.Errorf(fmt.Sprintf("Billing account is invalid. Run the command again with a valid billing account. %s",
```

Comment:
@gdbranco Seems like the fmt.Errorf was complaining while r.Reporter.error was not. I was seeing something like the screenshot. Adding a fmt.Errorf(fmt.Sprintf()) fixes the issue in validateNetworkType.
<img width="809" alt="image" src="https://github.com/openshift/rosa/assets/26282541/54fd2677-25b1-49e3-a154-229e3cf0744e">

Reply:
@chenz4027 its a best practice warning. If a func is returning an error it should not be capitalized, as it can be wrapped which can lead to some weird formatting.

Reply:
Ah got it, @ciaranRoche will update the change

Reply:
@ciaranRoche @gdbranco Would you mind taking another look? 🙏 Hope it could make it to this rosa release. If not, I'll wait for the next one then.